<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>748</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-color: rgb(16, 255, 24);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="GameControl">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>701</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="FLoading">
     <widget class="QLabel" name="SunFlow">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>220</y>
        <width>71</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>230</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 25pt &quot;Tw Cen MT Condensed&quot;;
color: rgb(80, 121, 37);
font: 25pt &quot;Gill Sans Ultra Bold Condensed&quot;;</string>
      </property>
      <property name="text">
       <string>Loading</string>
      </property>
     </widget>
     <widget class="QLabel" name="points">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>230</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 25pt &quot;Tw Cen MT Condensed&quot;;
color: rgb(80, 121, 37);
font: 25pt &quot;Gill Sans Ultra Bold Condensed&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="LoginPage">
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>340</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Display Semibold</family>
        <pointsize>16</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(104, 101, 91);
color: rgb(255, 255, 255);
font: 600 16pt &quot;Sitka Display Semibold&quot;;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>410</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Display Semibold</family>
        <pointsize>16</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(104, 101, 91);
color: rgb(255, 255, 255);
font: 600 16pt &quot;Sitka Display Semibold&quot;;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>340</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tw Cen MT Condensed Extra Bold</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(4, 101, 3);
color: rgb(80, 121, 37);</string>
      </property>
      <property name="text">
       <string>Username : </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>410</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tw Cen MT Condensed Extra Bold</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(4, 101, 3);
color: rgb(80, 121, 37);</string>
      </property>
      <property name="text">
       <string>Password :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Ok">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>470</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(104, 101, 91);
color: rgb(83, 255, 212);</string>
      </property>
      <property name="text">
       <string>Ok</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SignUp">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>511</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 137, 145);
background-color: rgb(104, 101, 91);</string>
      </property>
      <property name="text">
       <string>SignUp</string>
      </property>
     </widget>
     <widget class="QPushButton" name="eye_login">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>410</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Signup">
     <widget class="QPushButton" name="Back">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>540</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>180</y>
        <width>263</width>
        <height>361</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="lineEdit_3">
         <property name="font">
          <font>
           <family>Sitka Display Semibold</family>
           <pointsize>16</pointsize>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(8, 199, 46);
background-color: rgb(80, 121, 37);
color: rgb(5, 5, 255);
font: 600 16pt &quot;Sitka Display Semibold&quot;;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_4">
         <property name="font">
          <font>
           <family>Sitka Display Semibold</family>
           <pointsize>16</pointsize>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(8, 199, 46);
background-color: rgb(80, 121, 37);
color: rgb(5, 5, 255);
font: 600 16pt &quot;Sitka Display Semibold&quot;;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_5">
         <property name="font">
          <font>
           <family>Sitka Display Semibold</family>
           <pointsize>16</pointsize>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(8, 199, 46);
background-color: rgb(80, 121, 37);
color: rgb(5, 5, 255);
font: 600 16pt &quot;Sitka Display Semibold&quot;;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_6">
         <property name="font">
          <font>
           <family>Sitka Display Semibold</family>
           <pointsize>16</pointsize>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(8, 199, 46);
background-color: rgb(80, 121, 37);
color: rgb(5, 5, 255);
font: 600 16pt &quot;Sitka Display Semibold&quot;;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_7">
         <property name="font">
          <font>
           <family>Sitka Display Semibold</family>
           <pointsize>16</pointsize>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(8, 199, 46);
background-color: rgb(80, 121, 37);
color: rgb(5, 5, 255);
font: 600 16pt &quot;Sitka Display Semibold&quot;;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_8">
         <property name="font">
          <font>
           <family>Sitka Display Semibold</family>
           <pointsize>16</pointsize>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(8, 199, 46);
background-color: rgb(80, 121, 37);
color: rgb(5, 5, 255);
font: 600 16pt &quot;Sitka Display Semibold&quot;;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>190</y>
        <width>202</width>
        <height>331</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">color: rgb(3, 218, 211);
font: 700 24pt &quot;Tw Cen MT Condensed&quot;;
background-color: rgb(12, 48, 2);</string>
         </property>
         <property name="text">
          <string>Full-Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="styleSheet">
          <string notr="true">color: rgb(3, 218, 211);
font: 700 24pt &quot;Tw Cen MT Condensed&quot;;
background-color: rgb(12, 48, 2);</string>
         </property>
         <property name="text">
          <string>username : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string notr="true">color: rgb(3, 218, 211);
font: 700 24pt &quot;Tw Cen MT Condensed&quot;;
background-color: rgb(12, 48, 2);</string>
         </property>
         <property name="text">
          <string>PhoneNumber : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="styleSheet">
          <string notr="true">color: rgb(3, 218, 211);
font: 700 24pt &quot;Tw Cen MT Condensed&quot;;
background-color: rgb(12, 48, 2);</string>
         </property>
         <property name="text">
          <string>Email_Address : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="styleSheet">
          <string notr="true">color: rgb(3, 218, 211);
font: 700 24pt &quot;Tw Cen MT Condensed&quot;;
background-color: rgb(12, 48, 2);</string>
         </property>
         <property name="text">
          <string>Password : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="styleSheet">
          <string notr="true">color: rgb(3, 218, 211);
font: 700 24pt &quot;Tw Cen MT Condensed&quot;;
background-color: rgb(12, 48, 2);</string>
         </property>
         <property name="text">
          <string>confirm password : </string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="SignupCheck">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>550</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Sign up</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>827</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
